db/migrate/20160830110050_change_cloud_subnets_network_ports_unique_index.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeCloudSubnetsNetworkPortsUniqueIndex < ActiveRecord::Migration[5.0]
+  def change
+    remove_index :cloud_subnets_network_ports,
+                 :column => [:cloud_subnet_id, :network_port_id],
+                 :name   => 'index_cloud_subnets_network_ports',
+                 :unique => true
+
+    add_index :cloud_subnets_network_ports, :address
+
+    add_index :cloud_subnets_network_ports,
+              [:cloud_subnet_id, :network_port_id, :address],
+              :name   => 'index_cloud_subnets_network_ports_address',
+              :unique => true
```

Comment:
I'll merge this since it's already an index that exists, but I'm wondering if we should have this unique index at all, and just do it with Rails model validations.  That way fixes can be backported (as is, this can't be backported, and this fix is not possible on darga at all).

Also, what problem is this unique index solving?  I would think that this information comes from the provider, so if they change their rules, why would we care?